src/api/index.js

```diff
@@ -88,11 +93,22 @@ export function getResourcesAPI(
   ].join('');
 }
 
-export function getTektonAPI(type, { name = '', namespace } = {}, queryParams) {
-  return getResourcesAPI(
-    { group: 'tekton.dev', version: 'v1alpha1', type, name, namespace },
-    queryParams
+export function getTektonAPI(
+  type,
+  { name = '', namespace } = {},
+  queryParams,
+  version
+) {
+  let versionToUse = 'v1beta1';
+  if (version) {
+    versionToUse = version;
+  }
+  const resourceAPI = getResourcesAPI(
+    { group: 'tekton.dev', type, name, namespace },
+    queryParams,
+    versionToUse
```

Comment:
```suggestion
export function getTektonAPI(
  type,
  { name = '', namespace, version = 'v1beta1' } = {},
  queryParams
) {
  const resourceAPI = getResourcesAPI(
    { group: 'tekton.dev', type, version, name, namespace },
    queryParams
```

we can set the default version in the first param and avoid the extra juggling here
then just pass it down to `getResourcesAPI` which would remain unchanged (as-is on master)